scripts/build_scripts/build_with_profiler_opt.sh

```diff
@@ -12,6 +12,14 @@ if [[ -z "$ARCH_NAME" ]]; then
 fi
 
 remove_default_log_locations
-make clean
-make build ENABLE_PROFILER=1 ENABLE_TRACY=1
-make programming_examples
+# make clean
+# make build ENABLE_PROFILER=1 ENABLE_TRACY=1
+# make programming_examples
+
+rm -rf build
+# ENABLE_TRACY=1 ENABLE_PROFILER=1 cmake -B build
+PYTHON_ENV_DIR=$(pwd)/build/python_env ENABLE_TRACY=1 ENABLE_PROFILER=1 ./scripts/build_scripts/create_venv.sh.sh -C $CONFIG
```

Comment:
`create_venv.sh.sh`  double extension

Reply:
thank you, cleanup mistake when I moved it to `scripts/build_scripts`